docs/deployment/ecs-weave-shippable.md

```diff
@@ -54,11 +56,11 @@ except for the front-end service.
 
 3. Select `Create Stack`
 
-4. Under `Choose a template`, select `Choose file`
+4. Under `Choose a template`, select `Choose file` or `Browse`
 
 5. Navigate to the directory where you cloned the <span style="color: orange">
 microservices-demo</span> repo
-and choose to upload the file `./deploy/aws-ecs/cloudformation.json`
+and choose to upload the file `./deploy/aws-ecs-shippable/cloudformation.json`
```

Comment:
Let's just add shippable to `.travis.yml` and publish to S3 bucket along with the main template, so that we can replace all these steps with just one button.